apps/showcase/scripts/copy-training-assets/index.cjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copy necessary assets for the Otter training
+ */
+const fs = require('node:fs');
+const path = require('node:path');
+
+const temporaryAssetsDirectory = path.join(process.cwd(), 'training-assets');
+
+function getAbsolutePath(packageName) {
+  const packageJsonPath = require.resolve(`${packageName}/package.json`);
+  return packageJsonPath.substring(0, packageJsonPath.lastIndexOf('\\'));
+}
+
+// @vscode/codicons
+const vscodePath = path.join(getAbsolutePath('@vscode/codicons'), 'dist');
+const vscodeDestinationPath = `${temporaryAssetsDirectory}/@vscode/codicons/dist`;
+fs.cpSync(vscodePath, vscodeDestinationPath, {recursive: true});
+
+// @xterm/xterm
+const xtermPath = path.join(getAbsolutePath('@xterm/xterm'), 'css/xterm.css');
+const xtermDestinationPath = `${temporaryAssetsDirectory}/@xterm/xterm/css/xterm.css`;
```

Comment:
If we only copy the css file here, we have no guarantee that if the xterm.css is updated in a new release, then it will still work. For instance, if it then refers to a relative file, the link will be broken.

Reply:
It might be more acceptable than unplugging all the import and having to override the project json to target the external packages everytime they are bumped